newIDE/app/src/ObjectEditor/Editors/SpriteEditor/AnimationPreview.js

```diff
@@ -289,20 +281,20 @@ const AnimationPreview = ({
                 }}
                 id="direction-time-between-frames"
                 type="number"
-                step={1}
-                min={1}
+                min={0}
                 max={100}
                 style={styles.timeField}
                 autoFocus="desktop"
               />
               <Timer style={styles.timeIcon} />
-              <TextField
+              <SemiControlledTextField
+                commitOnBlur
                 margin="none"
-                value={timeBetweenFramesRef.current}
-                onChange={(e, text) => {
-                  const time = parseFloat(text);
+                value={timeBetweenFramesRef.current.toString()}
+                onChange={text => {
+                  if (!text) return;
+                  const time = Number.parseFloat(text);
                   if (time > 0) {
-                    setFps(Math.round(1 / time));
```

Comment:
so, this is not needed anymore because updating the timebetweenframesref also updates the fps in the next render?

Reply:
Well, it changes the direction property `timeBetweenFrames` and the parent component changes force updates, changing the props timeBetweenFrames given to AnimationPreview so fps is up to date